src/mockConverter.ts

```diff
@@ -69,34 +69,81 @@ function getStringFakeValue({
-        return schema[SwaggerProps.AllOf]
+const getIsSchemaContainsAllOfArray = (schema:any) => {
+    return schema && schema[SwaggerProps.AllOf] && Array.isArray(schema[SwaggerProps.AllOf]);
+}
+
+/**
+ * Get all interfaces that this schema exteds
+ * @param schema DTO schema
+ * @param DTOs all DTOs
+ */
+export const getSchemaInterfaces = (schema: any, DTOs: any): Array<string> | undefined => {
+
+    if (getIsSchemaContainsAllOfArray(schema)) {
+        const result = [] as Array<string>;
+
+        schema[SwaggerProps.AllOf]
             .filter((e: { $ref?: string }) => e[SwaggerProps.$ref])
-            .map((obj: any) => {
+            .forEach((obj: any) => {
                 const refType = obj[SwaggerProps.$ref].split('/');
-                return parseRefType(refType);
+                /*
+                Example: will return InviteMembersRequestDto
+                if "SomeDTo extends InviteMembersRequestDto"
+                 */
+                const parsedRefType = parseRefType(refType);
+
+                // Repeat "getSchemaInterfaces" in cycle for inner interfaces
+                const newSchema = DTOs[parsedRefType];
+                if (getIsSchemaContainsAllOfArray(newSchema)) {
+                    getSchemaInterfaces(newSchema, DTOs)?.forEach(b => {
+                        result.push(b);
+                    });
+                } else {
+                    if (newSchema) {
+                        result.push(parsedRefType);
+                    }
+                }
+
+                result.push(parsedRefType);
             });
+
+        return result;
     } else {
         return undefined;
     }
 };
 
-export const combineProperties = ({ schema, schemas, interfaces }: any) => {
-    if (interfaces) {
-        let properties = Object.assign({}, schema.properties);
-        interfaces.map((interfaceName: string) => {
-            const dto = schemas[interfaceName];
-            if (dto && dto.properties) {
-                properties = Object.assign(properties, dto.properties);
-            }
-        });
+interface CombinePropertiesProps {
+    schema: any;
+    schemas: any;
+    interfaces: Array<string>;
+}
 
-        schema.properties = properties;
-        return Object.assign({}, schema);
-    } else {
-        return schema;
-    }
+/**
+ * Combines all properties from extended DTOs into one object.
+ *
+ * @param schema
+ * @param schemas
+ * @param interfaces
+ */
+export const combineProperties = ({ schema, schemas, interfaces }: CombinePropertiesProps) => {
+    let properties = Object.assign({}, schema.properties);
+
+    interfaces.forEach((interfaceName: string) => {
+        const dto = schemas[interfaceName];
+
+        if (dto && dto.properties) {
+            properties = Object.assign(properties, dto.properties);
+        }
+
+        if (dto && dto[SwaggerProps.AllOf] && dto[SwaggerProps.AllOf][1].properties) {
```

Comment:
As you are reassigning `properties`, it's probably better to do `else if`

However can it happen that both `dto.properties` and `dto[SwaggerProps.AllOf][1].properties` exist?
If so should you merge all `Object.assign`?

Btw you could use spread operator as well instead of `Object.assign`